languages/common-lisp/exercises/concept/pizza-pi/.docs/approaches.md

```diff
@@ -0,0 +1,18 @@
+### A More Idiomatic Approach
```

Comment:
ah the approaches document. later i'll want to review the other exercises - maybe we should have some more 'apporaches'.

Reply:
Yeah, I think it's a quite placeholder document for the time being, but I've added this here just because it was already part of the `after.md` :)